app-engine-exec-wrapper/execute.sh

```diff
@@ -97,10 +100,15 @@ if [ -n "${SQL_INSTANCES}" ]; then
   fi
 fi
 
+if [ -n "${ENTRYPOINT}" ]; then
+  ENTRYPOINT="--entrypoint ${ENTRYPOINT}"
```

Comment:
I'd guess that, beyond Procfiles, it might be common for an entrypoint to include spaces. I'd suggest adding quotes here so that a construct like `-r "bundle exec ruby"` will work.